.github/workflows/graalvm.yml

```diff
@@ -21,6 +21,9 @@ jobs:
       matrix:
         java: ['17']
         graalvm: ['latest', 'dev']
```

Comment:
@alvarosanchez Is this the change you suggested?

Reply:
In theory, yes, but I tried [here](here) and it still generated 3 jobs:

![image](https://user-images.githubusercontent.com/153880/214242190-5a333f16-58f1-465c-8469-9e76981f2e6e.png)

I will try with `exclude` instead.

Reply:
This did the trick:

```
      matrix:
        graalvm: ['latest', 'dev']
        java: ['17']
        include:
          - graalvm: 'dev'
            java: '19'
        exclude:
          - graalvm: 'dev'
            java: '17'
```

Reply:
Updated